### DISCUSSION TITLE
PWM RC Speed-controler

### DISCUSSION THREAD

**Discussion by wollo-fpv** [arendst/Tasmota#15516] [Q&A] [answered]
Hello,
I have a DC 12V/120Watt brushed fan.
I want to control the rpm of the fan. 
10 Amps at start up even a higher current, I thought to user a RC speed-controller
https://hobbyking.com/de_de/quicrun-wp-1625-brushed.html?queryID=00de9d659b39820c655208f20e47485e&objectID=62437&indexName=hbk_live_products_analytics
Until now I have user tasmota only to read temperature and to switch on and off devices.
I have no idea how to start....
And if t will be possible...
I will be happy for any help..... :)

Wolf

**Comment by barbudor** [comment, marked as answer]
Hi
From what I read [here](https://www.hobbywing.com/goods.php?id=359) it looks like the ESC is controlled by a servo-like PWM control from a RC receiver.
Tasmota doesn't have a servo control output mode but it seems that servos can still work with Tasmota PWM output as per : https://github.com/arendst/Tasmota/discussions/10443

That may give you some hints on how to control the speed with Tasmota.

**Reply by wollo-fpv** [reply]
Yes it is.....
Will give it a try
Thx

**Comment by Jason2866** [comment]
It works, i did such a setup.. Replace the Tamiya connector. It is not good for permanent high current.

**Reply by wollo-fpv** [reply]
I use a XT60 connector!

**Comment by wollo-fpv** [comment]
Thx,
It's working fine.... :) :)
Wolf
